tests/vmcheck/test-layering-scripts.sh

```diff
@@ -77,6 +77,25 @@ vm_cmd cat /usr/lib/prefixtest.txt > prefixtest.txt
 assert_file_has_content prefixtest.txt "/usr"
 echo "ok script expansion"
 
+vm_rpmostree rollback
+vm_reboot
+vm_rpmostree cleanup -p
+vm_build_rpm scriptpkg4 \
+             transfiletriggerin "/usr/bin" 'cat >/usr/share/transfiletriggerin-bin.txt'
+vm_build_rpm scriptpkg5 \
+             transfiletriggerin "/usr/lib" 'cat >/usr/share/transfiletriggerin-lib.txt'
```

Comment:
We should have a third one as well which has both patterns to make sure we can handle a trigger with multiple patterns.